### PR TITLE
PAN-OS -  static routes + pcaps

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/20400
fixes: https://github.com/demisto/etc/issues/20854
fixes: https://github.com/demisto/etc/issues/20898

## Description
add static route CRUD for virtual router network object.
can be adde to the FW network, or to a panorama template.

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests - TPB and unit tests.
- [ ] Documentation (with link to it)
- [x] Code Review

@mayagoldb FYI

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](https://github.com/demisto/content/blob/fb0e63afbba8c8add4deebc42194c66b04ac5c21/Integrations/Panorama/Panorama.yml)
- [ ] [CHANGELOG](https://github.com/demisto/content/blob/fb0e63afbba8c8add4deebc42194c66b04ac5c21/Integrations/Panorama/CHANGELOG.md)

### CODE REVIEW

**Comment by bakatzir** [comment]
@yuvalbenshalom 
Screen Shot 2019-12-19 at 13 50 07
Screen Shot 2019-12-19 at 17 51 23

File: Integrations/Panorama/Panorama.py [hunk 1]
```diff
@@ -55,6 +55,21 @@
...
+if TEMPLATE:
+    if not DEVICE_GROUP or VSYS:
+        return_error('Template is only relevant for Panorama instances.')
+# setting network xpath relevant to FW or panorama management
+if DEVICE_GROUP:
+    XPATH_NETWORK = f'/config/devices/entry[@name=\'localhost.localdomain\']/template/entry[@name=\'{TEMPLATE}\']' \
+                    f'/config/devices/entry[@name=\'localhost.localdomain\']/network'
+else:
+    XPATH_NETWORK = "/config/devices/entry[@name='localhost.localdomain']/network"
+
```
Comment (yuvalbenshalom) [inline Integrations/Panorama/Panorama.py]: Accessing the arguments(apposed to parameters) in the beginning of the integration is confusing and anti-pattern. I think this code should be moved to a single function that gets the `template` argument(from args or params) and turns the XPATH_NETWORK. this function will be used by the relevant business logic functions that needs the XPATH_NETWORK value.

**Comment by richardbluestone** [comment]
Docs reviewed and updated

**Comment by bakatzir** [comment]
@richardbluestone No changes in the yml? are the new commands outputs and args descriptions ok?

**Comment by richardbluestone** [comment]
I added the changes - not sure what happened before

**Pull request merged**